querybook/server/datasources/datadoc.py

```diff
@@ -317,6 +317,10 @@ def update_datadoc_schedule(id, cron=None, enabled=None, kwargs=None):
         api_assert(schedule, "Schedule does not exist")
         verify_data_doc_permission(id, session=session)
 
+        # schedule update will not change the owner
```

Comment:
in this case, let's just remove / ignore this field and get the owner_uid from datadoc